nsv13/code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -0,0 +1,26 @@
+/datum/gas/constricted_plasma
+	id = "constricted_plasma"
+	specific_heat = 250
+	name = "Constricted plasma"
+	gas_overlay = "constricted_plasma"
+	moles_visible = MOLES_GAS_VISIBLE
+	dangerous = TRUE
+	rarity = 1000
+
+/datum/gas/nucleium
+	id = "nucleium"
+	specific_heat = 850
+	name = "Nucleium"
+	gas_overlay = "nucleium"
+	moles_visible = MOLES_GAS_VISIBLE
+	dangerous = TRUE
+	rarity = 2000
+
+/datum/gas/frameshifted_plasma
```

Comment:
As previously discussed, please do not add an additional gas datum purely for this device.

We have Nucleium for this express purpose.